### DISCUSSION TITLE
Remapping keyboard keys from the host to the Amiga

### DISCUSSION THREAD

**Discussion by caleb** [BlitterStudio/amiberry#1240] [Q&A]
Hi,

I was wondering if it was possible to remap keys from my host keyboard to the Amiga's keyboard. Specifically I want to map WASD to the arrow keys or keypad keys.

I tried putting this:

keyboard_key_q=action_key_np_7
keyboard_key_w=action_key_cursor_up
keyboard_key_e=action_key_np_9
keyboard_key_a=action_key_cursor_left
keyboard_key_s=action_key_cursor_down
keyboard_key_d=action_key_cursor_right

In a config file that I am loading in the "Configurations" directory, but this has no effect (I got these instructions from FS-UAE.

Am I putting them in the wrong place or does Amiberry not map keys this way?

Thanks!
Caleb

**Comment by midwan** [comment]
That kind of config is specific to FS-UAE, and it won't work I'm afraid. 
There isn't a keyboard mapping feature in Amiberry in that sense. But we do have some predefined mappings that use the keyboard as a joystick, instead - you may find that as a good enough alternative?
